SpatialGDK/Source/SpatialGDK/Public/SpatialCommonTypes.h

```diff
@@ -49,5 +49,6 @@ struct FTrackableWorkerType : public T
 #endif
 };
 
+// TODO: These can be removed once event tracing is enabled UNR-3981
```

Comment:
Is this meant as a follow up task?

Reply:
This will be removed as part of https://improbableio.atlassian.net/jira/software/c/projects/UNR/issues/UNR-3981